docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -32,7 +32,7 @@ When a FILTER clause is attached to an aggregate function, only the matching row
 GROUP BY group_expression [ , group_expression [ , ... ] ]
     [ { WITH ROLLUP | WITH CUBE | GROUPING SETS (grouping_set [ , ...]) } ]
```

Comment:
BTW do we really support `GROUP BY a, b GROUPING SETS (c, d)`?

Reply:
> BTW do we really support `GROUP BY a, b GROUPING SETS (c, d)`?

Not yet. Same reason as https://github.com/apache/spark/pull/30212#discussion_r601171306

Reply:
maybe we should change this to
```
GROUP BY group_expression [ , group_expression [ , ... ] ]
    [ { WITH ROLLUP | WITH CUBE } ]
```

`GROUP BY GROUPING SETS (grouping_set [ , ...])` is covered by the syntax below.

Reply:
You can see the comment https://github.com/apache/spark/pull/30212/files#r529501391

Not all case can be covered such as `GROUP BY A, B grouping sets(a, (a, b ))`